packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -262,7 +262,7 @@ export class UIFormComponent extends React.Component {
 				/>
 			));
 		const buttonsRenderer = () => (
-			<div className={classNames(theme['form-actions'], 'tf-actions-wrapper')}>
+			<div className={classNames(theme['form-actions'], 'tf-actions-wrapper')} key="form-buttons">
```

Comment:
since buttonsRenderer is called multiple time it should have a unique key associated with each execution in sequence.

https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
for me it s not the case it s called at each render but it identify the form submit item. So it s identity is form-buttons

It s not the widget.

![screen shot 2018-11-13 at 11 58 40](https://user-images.githubusercontent.com/19857479/48409278-add05b00-e73b-11e8-9826-18618a44467a.png)

Reply:
ok it is because buttonsRenderer is called inside an array, i think a comment on the code should be put, so we do not forget to migrate to React.Fragment later, because a static key is pretty counter intuitive here